plugins/dell/dell.quirk

```diff
@@ -1,8 +1,12 @@
-[fwupd-uefi-version-format]
+[SmbiosManufacturer=Dell Inc.]
+UefiVersionFormat = quad
 
-# Dell & Alienware use AA.BB.CC.DD rather than AA.BB.CCDD
-Dell Inc.=none
-Alienware=none
+[SmbiosManufacturer=Alienware]
+UefiVersionFormat = quad
 
-[fwupd-daemon-version-format]
-com.dell.uefi*.firmware=none
+# DEPRECATED: put this in the AppStream metadata itself using
+# <custom>
+#   <value key="LVFS::VersionFormat">quad</value>
+# </custom>
+[DaemonVersionFormat=Quad]
+ComponentIDs = com.dell.uefi*.firmware
```

Comment:
Do you want to still see this land in the CAB file itself in the future or let LVFS do it on the OEM account? I would prefer to keep it in the OEM account.

Reply:
I don't think it matters as long as something is setting it in the metadata tag. I'm happy for this quirk to live here for a long time, but I don't really want to see any more vendors adding to the list.